src/injected/frameCommunicators/html-element-axe-results-helper.ts

```diff
@@ -22,12 +22,14 @@ export interface AssessmentVisualizationInstance extends AxeResultsWithFrameLeve
     propertyBag?: any;
 }
 
-export interface TabStopVisualizationInstance extends AssessmentVisualizationInstance {
-    requirementResults: {
-        [requirementId in TabStopRequirementId]: {
-            description: string;
-        };
+export type TabStopVisualizationRequirementResults = Partial<{
+    [requirementId in TabStopRequirementId]: {
+        instanceId: string;
```

Comment:
Changed this to use instanceId instead because I don't see a use case for description but instance id would be useful for if/when we implement the scroll-to-instance stretch goal.